{{ cookiecutter.project_name }}/docker/docker-compose.yml

```diff
@@ -21,9 +21,12 @@ services:
       - "127.0.0.1::5000"
     volumes:
       - ../:/mnt
-    entrypoint: bash -c "mlflow server -h 0.0.0.0 --backend-store-uri ${MLFLOW_TRACKING_URI} --default-artifact-root ${MLFLOW_ARTIFACT_LOCATION:-'None'} -p 5000 && /bin/bash"
+    entrypoint: bash -c "mlflow server -h 0.0.0.0 --backend-store-uri $${MLFLOW_TRACKING_URI} --default-artifact-root $${MLFLOW_ARTIFACT_LOCATION:-'None'} -p 5000 && /bin/bash"
     stdin_open: true
     container_name: "{{ cookiecutter.package_name }}_mlflow_${USER}"
     tty: true
+    environment: # takes precedent over env file
+      - MLFLOW_TRACKING_URI
```

Comment:
Does this fall back to the .env file if these variables aren't set?

Reply:
Yep!